db/models.py

```diff
@@ -14,3 +14,44 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
The `related_name` for the `genres` field should be unique across the application. Using 'genres' as the `related_name` might cause conflicts if there are other ManyToMany fields pointing to the `Genre` model. Consider using a more specific name like 'movie_genres'.